src/everest/config/control_config.py

```diff
@@ -49,124 +50,171 @@ class ControlConfig(BaseModel):
+            """
+            Enable/disable control groups.
+
+            Whether all control variables in the control group are enabled or
+            not. When not enabled, variables are kept constant at the intial
+            guess value during optimization. This can be overridden at the
+            variable level.
+            """
+        ),
     )
     min: float | None = Field(
         default=None,
-        description="""
-Defines left-side value in the control group range [min, max].
-This value will be overwritten by the control variable min value if given.
-
-The initial guess for both the group and the individual variables needs to be contained
-in the resulting [min, max] range
-""",
+        description=dedent(
+            """
+            The minimum values of all control variables in the control group
+            unless overridden at the variable level.
+
+            The `initial_guess` field of this control group and of all of its
+            variables must respect this minimum value.
+            """
+        ),
     )
     max: float | None = Field(
         default=None,
-        description="""
-Defines right-side value in the control group range [min, max].
-This value will be overwritten by the control variable max value if given.
-
-The initial guess for both the group and the individual variables needs to be contained
-in the resulting [min, max] range
-""",
+        description=dedent(
+            """
+            The maximum values of all control variables in the control group
+            unless overridden at the variable level.
+
+            The `initial_guess` field of this control group and of all of its
+            variables must respect this maximum value.
+            """
+        ),
     )
     perturbation_type: Literal["absolute", "relative"] = Field(
         default="absolute",
-        description="""
-Example: absolute or relative
-Specifies the perturbation type for a set of controls of a certain type.  The
- perturbation type keyword defines whether the perturbation magnitude
- (perturbation_magnitude) should be considered as an absolute value or relative
- to the dynamic range of the controls.
-
-NOTE: currently the dynamic range is computed with respect to all controls, so
- defining relative perturbation type for control types with different dynamic
- ranges might have unintended effects.
-        """,
+        description=dedent(
+            """
+            The perturbation type for the control group.
+
+            The `perturbation_type` keyword defines whether the perturbation
+            magnitude (`perturbation_magnitude`) should be treated as an
+            absolute value or as relative to the dynamic range of the controls.
+
+            Note: currently the dynamic range is computed with respect to all
+            controls, so defining a relative perturbation type for control types
+            with different dynamic ranges might have unintended effects.
```

Comment:
Is this something Everest could warn when simulating? Perhaps we already do?

Reply:
I checked what is happening in the code, and this comment does not make sense in this context. It may refer to the way the default value for `perturbation_magnitude` is calculated. And I noticed now that the approach for calculating that default is pretty bad. To my knowledge, current practice (e.g. in the tutorials) is to always manually set this. We should probably make `perturbation_magnitude` mandatory.

Reply:
There is an issue about the `perturbation_magnitude`, (#9844) I will amend.